src/reporters/json.ts

```diff
@@ -16,20 +16,39 @@
 
 import fs from 'fs';
 import path from 'path';
+import os from 'os';
 import { Config } from '../config';
 import { EmptyReporter } from '../reporter';
+import { spawnSync } from 'child_process';
 import { Test, Suite, Spec, TestResult, TestError } from '../test';
 
 export interface SerializedSuite {
   title: string;
   file: string;
-  location: string,
+  location: string;
   specs: ReturnType<JSONReporter['_serializeTestSpec']>[];
   suites?: SerializedSuite[];
 }
 
+type CommitInfo = {
```

Comment:
Do you think you can collect this information separately? It looks ok, but I am not sure I want to commit to it for life...